### PR TITLE
feat(distplot): add property in axis simple mode

### PR DESCRIPTION
**Pull request by jjxie**
https://user-images.githubusercontent.com/25456307/168547228-086e605c-0714-4e5d-a605-54af94323382.mov

### CODE REVIEW

File: charts/distributionplot/src/distributionplot-properties.js [hunk 1]
```diff
@@ -318,6 +318,52 @@ export default function propertyDefinition(env) {
 
   const measureAxis = {
     uses: 'axis.picasso.measureAxis',
+    items: {
+      startAt: {
```
Comment (T-Wizard) [inline charts/distributionplot/src/distributionplot-properties.js:322]: is the definition of startAt the same in all charts that have it
maybe add it in default-properties in sense-client to support 

```
startAt: {
   use: 'startAt'
}
```

in the charts that needs it

File: chart-modules/common/locale/all.json [hunk 2]
```diff
@@ -564,5 +564,23 @@
       "ja-JP": "99 番目の百分位数",
       "ru-RU": "99-й процентиль"
     }
+  },
+  "properties.axis.startAt": {
```
Comment (T-Wizard) [inline chart-modules/common/locale/all.json:568]: there should be no need to add properties strings here as they are only used inside sense-client

File: charts/distributionplot/src/distributionplot-properties.js (hunk 1 above)
Comment (jjxie) [reply charts/distributionplot/src/distributionplot-properties.js:322]: start at added to chart that are not bar type, for example bar chart and histogram won't have it. Also the autoMinMax, minMax, min are not in same location, for example combo chart, line chart and scatterplot.

File: chart-modules/common/locale/all.json (hunk 2 above)
Comment (jjxie) [reply chart-modules/common/locale/all.json:568]: Ok, I will remove it

**Review by T-Wizard — APPROVED** [review]

**Pull request merged**